.github/workflows/windows-release.yml

```
@@ -0,0 +1,157 @@
+          Write-Host "Signature Status: $($sig.Status)"
+          if ($sig.Status -ne "Valid") { throw "Signature verification failed: $($sig.StatusMessage)" }
+
+          # Create final filename with version
+          $finalExeName = "giddh Setup $env:PACKAGE_VERSION.exe"
+          "FINAL_EXE_NAME=$finalExeName" | Out-File -FilePath $env:GITHUB_ENV -Append
+          Write-Host "Final EXE name: $finalExeName"
+
+      # --- AWS Auth (choose ONE path: OIDC role OR access keys) ---
+
+      - name: Configure AWS (OIDC role)
+        if: secrets.AWS_ROLE_TO_ASSUME != ''
```

actionlint (1.7.10)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)